improver/metadata/amend.py

```diff
@@ -66,6 +68,22 @@ def update_stage_v110_metadata(cube: Cube) -> None:
     cube.attributes["mosg__grid_version"] = "1.1.0"
 
 
+def _load_json(json_path: str) -> Dict:
+    """Load a JSON file into a dictionary.
+
+    Args:
+        json_path:
+            Path to the JSON file to load.
+
+    Returns:
+        Dictionary containing the JSON data.
+    """
+    if not isinstance(json_path, str):
+        return json_path
+    json_path = os.path.expanduser(os.path.expandvars(json_path))
+    return json.load(open(json_path, "r"))
+
+
 def amend_attributes(cube: Cube, attributes_dict: Dict[str, Any]) -> None:
```

Comment:
```suggestion
def amend_attributes(cube: Cube, attributes_dict: Union[str, Dict[str, Any]]) -> None:
```
And the doc-string needs updating to show that `attributes_dict` can be a string (or Path object?) pointing to a JSON file.